journal/modernized/20-chapter-17.adoc

```diff
@@ -353,8 +353,8 @@ The bulwark on the starboard side was damaged,
 and the spray reached more than two-thirds up the main-mast.
 In the morning, the seamen expressed considerable discouragement;
 and I observed a disposition rather to make the worst of things.
-The captain said,
-'he hoped he should not see such another night in this vessel.' One
+The captain said
+he hoped he should not see such another night in this vessel. One
```

Comment:
This didn't seem like a direct quote to me.

Reply:
Yup, agreed. In these cases it's a bit of a judgment call. Removing the single quotes is an option, or sometimes we use italics to indicate an "indirect quote," since that's a convention that these old books often used themselves.  But in this case, I think I agree with just nixing the single quotes instead of reaching for italics.